messages/views.py

```diff
@@ -181,6 +178,14 @@ def new_message(request, username=None, message_id=None):
     return render(request, 'messages/new.html', tvars)
 
 
+def quote_message_for_reply(body, username):
+    body = ''.join(BeautifulSoup(body).find_all(text=True))
```

Comment:
bs4 also has a `get_text()` method, but it won't return the text inside a `<!--comment-->`, whereas `find_all(text=True)` will. Do we prefer any specific method?

Reply:
I don’t think we have preference. If this works as expected we’re fine.

Reply:
OK, so let's leave it as-is for now.
I notice that when people write a message we actually send it through bleach (`utils/text.py::clean_html`). The only other place that we use bs4 is to filter http `<img>` tags and turn them into an anchor (in order to prevent cross-protocol errors). We could probably use bleach/html5lib for this too, and remove another dependency.